src/main/java/graphql/schema/idl/RuntimeWiring.java

```diff
@@ -69,6 +69,26 @@ public static Builder newRuntimeWiring() {
         return new Builder();
     }
 
+    /**
+     * @return a builder of Runtime Wiring based on the provided one
+     */
+    public static Builder newRuntimeWiring(RuntimeWiring original) {
+        Builder builder = new Builder();
+        builder.dataFetchers.putAll(original.dataFetchers);
+        builder.defaultDataFetchers.putAll(original.defaultDataFetchers);
+        builder.scalars.putAll(original.scalars);
+        builder.typeResolvers.putAll(original.typeResolvers);
+        builder.registeredDirectiveWiring.putAll(original.registeredDirectiveWiring);
+        builder.directiveWiring.addAll(original.directiveWiring);
+        builder.wiringFactory = original.wiringFactory;
+        builder.enumValuesProviders.putAll(original.enumValuesProviders);
+        builder.schemaGeneratorPostProcessings.addAll(original.schemaGeneratorPostProcessings);
+        builder.fieldVisibility = original.fieldVisibility;
+        builder.codeRegistry = original.codeRegistry;
+        builder.comparatorRegistry = original.comparatorRegistry;
+        return builder;
+    }
+
```

Comment:
Can we please have a test that shows some aspects of his being transferred

Reply:
also have a convention of a `transform` function from an existing object

For example `graphql.ExecutionInput#transform`

This would be something like (psudeo java)

```
public RuntimeWiring transform(Consumer<Builder> builderConsumer) {
        Builder builder =  newRuntimeWiring(this);

        builderConsumer.accept(builder);

        return builder.build();
    }
```


This allows some to take an existing object, transform it back into builder state and then manipulate it back into a new object

Can you please add `transform` and have a test for that please